src/Components/ArtistFilter/Artworks.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from "react"
+import {
+  ConnectionData,
+  createPaginationContainer,
+  graphql,
+  RelayPaginationProp,
+} from "react-relay"
+import { Subscribe } from "unstated"
+
+import { FilterState } from "./state"
+import ArtworkGrid from "../ArtworkGrid"
+
+interface Props extends RelayProps {
+  relay?: RelayPaginationProp
+  artistID: string
+}
+
+const PageSize = 10
+
+class Artworks extends React.Component<Props, null> {
+  loadMoreArtworks(filters) {
+    const hasMore = this.props.filtered_artworks.artworks.pageInfo.hasNextPage
+    const endCursor = this.props.filtered_artworks.artworks.pageInfo.endCursor
+    if (hasMore) {
+      // TODO: Should refetchConnection keep appending records?
+      this.props.relay.refetchConnection(
```

Comment:
It seems like this should keep re-initializing the connection, but in testing this seems to just keep appending records (so analogous to `loadMore`), leading to a hack to only display the current page of artworks.

We'll probably need to come back to this (good call at identifying that prev/next as well as arbitrary page number jumping is going to be difficult).

Reply:
@mzikherman - introducing the `paginationContainer` would fix this, yah?